tailwind.config.js

```diff
@@ -1,24 +1,29 @@
-const colors = require('tailwindcss/colors')
+const colors = require("tailwindcss/colors");
```

Comment:
@Gamaranto, jason mentioned this was a pain to configure tailwind. any tips on this and does this config match your expecations?

Reply:
Because I this it requires PostCSS there is a lot babel config / stuff that generally needs to happen it what is largely just a CSS lib.

Not sure there's much around it, just the way it works if we wanna use tailwind. So it requires that special babel config lib, or craco, using @jsxImport, etc.